### PR TITLE
About and Works sections 

### PR DESCRIPTION
**Pull request by Mahamadou-Nouridine**
In this Pr I've done these tasks:
- create **Works** section
- added a list of works in the section
- create **About Me** section
- added a brief introduction of myself
- added a list of **Languages Framworks** and **Skills**

### CODE REVIEW

File: about.css [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+
+.projects,
+.profile {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  padding: 0 24px;
+}
```
Comment (Tufahel) [inline about.css:130]: - [ ] Kindly use `display: grid` to the main container of the `works` section.  Though `projects` section is the main container but I can see `projects` section doesn't contain all the cards, rather `projects-card` section contains all cards more specifically. You can consider using a `grid` in the `projects-card` section and see the effect and then design according to it.

File: index.html [hunk 2]
```diff
@@ -60,13 +62,179 @@ <h3 class="headline-title2">I'm A Software Developer</h3>
...
+                </div>
+                <button class="card-button">See project</button>
+            </div>
+
+        </div>
+    </section>
+
+    <section class="profile">
+        <div class="about-me">
+            <h1 class="about-title">About me</h1>
```
Comment (Tufahel) [inline index.html]: - [ ] In a webpage it is not professional and SEO friendly if we use `h1` tag more than once. It is also not good for accessibility. You have already used `h1` tag for the title of the webpage. Kindly use other `hx` tags to fix this issue.

**Review by Tufahel — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @Mahamadou-Nouridine  :wave: 

Well done. :+1:  
You need to fix some issues before moving to the next project. :running_man: 

## Highlights 🌟

Alt Text

✔️ Linters checks are passing.
✔️ Readme is professional.


## Required Changes ♻️

> Kindly check the inline comments for review.

## Optional suggestions 🤷‍♂️

- ***[OPTIONAL]*** Consider using the capital letter as the first letter of a commit message, as it is professional.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.
- You can mail me at [Mail](mailto:tufahel96@gmail.com) or ask any question at Slack, `Name: Tufahel`

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by victorokoroji — APPROVED** [review]
Hi @Mahamadou-Nouridine, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Good UI design ✅ 
- The README is well documented ✅ 
- Good use of semantic HTML tags ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**